src/server/rdb_save.h

```diff
@@ -58,9 +58,11 @@ class AlignedBuffer : public ::io::Sink {
 
 // SaveMode for snapshot. Used by RdbSaver to adjust internals.
 enum class SaveMode {
-  SUMMARY,       // Save only header values (summary.dfs). Expected to read no shards.
-  SINGLE_SHARD,  // Save single shard values (XXXX.dfs). Expected to read one shard.
-  RDB,           // Save .rdb file. Expected to read all shards.
+  SUMMARY,                    // Save only header values (summary.dfs). Expected to read no shards.
```

Comment:
So the summary mode is not used anymore on its own. Please also not that we broke compatibility

Reply:
It's still used in snapshotting.

I'm not sure that this breaks compatability, this enum is not really serialized anywhere. Or do you mean we break compat for other reason?

Reply:
Ah, yes. You still have control over it. Compatibility with replication actually shouldn't break either because the rdb_loader in the shard thread will just load the scripts